### ISSUE TITLE
Unable to install the latest plugin version 2.0.5

### ISSUE THREAD

**Issue by famuyiwad** [logstash-plugins/logstash-input-kinesis#32]
I am having problem building a docker container of logstash with particular version, 2.0.5 of this plugin

Excerpt from my docker file
```
FROM logstash:5.4
RUN /usr/share/logstash/bin/logstash-plugin install --version 2.0.5 logstash-input-kinesis
```
And the error I am getting

> Validating logstash-input-kinesis-2.0.5
> Plugin logstash-input-kinesis version 2.0.5 does not exist
> ERROR: Installation aborted, verification failed for logstash-input-kinesis 2.0.5
> The command '/bin/sh -c /usr/share/logstash/bin/logstash-plugin install --version 2.0.5 logstash-input-kinesis' returned a non-zero code: 1

Is it not yet on the gem server or are mine missing something please?

Thanks in advance.

**Comment by famuyiwad** [comment]
I just had a look here, https://rubygems.org/gems/logstash-input-kinesis

2.0.5 is not there.

Can someone help please? I would really appreciate an early response because I have a dependency in a story I am currently working in our sprint that will end tomorrow to use profile authentication.

**Comment by famuyiwad** [comment]
Oh I see, it does look as if 2.0.5 release hasn't been done yet
https://github.com/logstash-plugins/logstash-input-kinesis/releases

Any idea when this will be done please?

**Comment by codekitchen** [comment]
I just published version 2.0.5, thanks
